packages/dicomImageLoader/examples/bootstrap.min.css

```diff
@@ -0,0 +1,6 @@
+/*!
```

Comment:
This should NOT be included here, but should come from node modules.  Please remove.